magefiles/mage.go

```diff
@@ -95,6 +96,31 @@ func (View) Docs() error {
 	return sh.Run("godoc")
 }
 
+// Clean is used to clean build results.
+func Clean() error {
+	fmt.Println("Cleanup ./bin...")
+	if err := sh.Run("rm", "-rf", "./bin"); err != nil {
+		return err
+	}
+	fmt.Println("Cleanup ./dist...")
+	if err := sh.Run("rm", "-rf", "./dist"); err != nil {
+		return err
+	}
+	// cleanup folders left after building snap in docker
+	// if folders do not exist - no problem, no error
+	fmt.Println("Cleanup snapcraft...")
+	if err := sh.Run("sudo", "rm", "-rf", "./parts", "./stage", "./prime"); err != nil {
```

Comment:
when `-f` is used I think no errors are reported.
Would it work to remove the `-f`, not to delete too much by accident?

Reply:
ok, agree, removed `-f`